packages/solid/src/generators/init/lib/add-dependencies.ts

```diff
@@ -0,0 +1,11 @@
+import { addDependenciesToPackageJson, Tree } from '@nrwl/devkit';
+
+export function updateDependencies(tree: Tree) {
+  const devDependencies = {
```

Comment:
The dependency 'vite-plugin-solid' is missing

Reply:
Nice :)